### PR TITLE
Feat/styling user profile

### PR DESCRIPTION
**Pull request by lee-cheung**
user profile styling

### CODE REVIEW

File: NativeApp/src/components/User/View/index.js
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/index.js]: I would move the ScrollView under HeaderDays component. This will keep the taken and remaining data at the top when scrolling.

So something like this:

```javascript 
<View style={styles.container}>
      <HeaderDays
        takenHolidays={takenHolidays}
        remainingHolidays={remainingHolidays}
      />
      <ScrollView>
        <View style={styles.profileName}>
....
```
Then add a background colour matching the header to the parent view. This will stop the white gap as well.

**Review by kirsty-unosquare — CHANGES_REQUESTED** [review]
Just need to remove the table and update the comments I mentioned.

File: NativeApp/src/components/User/View/index.js
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/index.js]: You can do `{`${employee.surname}'s Holidays`}`. Looks cleaner.

File: NativeApp/src/components/Common/HeaderDays/styles.js
Comment (kirsty-unosquare) [inline NativeApp/src/components/Common/HeaderDays/styles.js]: Would prefer if we removed `style-components` but for the time being add `marginTop: 50` to the `View`. The header is currently covering the options on iPhoneX.

File: NativeApp/src/components/User/View/ListItem.js
Comment (kirsty-unosquare) [inline NativeApp/src/components/User/View/ListItem.js]: I would change this to a normal list style rather than a table. It's currently very crushed looking. Maybe do something like the status on the booking screen.

**Review by kirsty-unosquare — CHANGES_REQUESTED** [review]

File: NativeApp/src/components/User/View/index.js
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: i have made the changes :)

File: NativeApp/src/components/User/View/index.js
Comment (lee-cheung) [reply NativeApp/src/components/User/View/index.js]: woops, completely forgot about template literals. yes, i have made the change

File: NativeApp/src/components/Common/HeaderDays/styles.js
Comment (lee-cheung) [reply NativeApp/src/components/Common/HeaderDays/styles.js]: done

File: NativeApp/src/components/User/View/ListItem.js
Comment (lee-cheung) [reply NativeApp/src/components/User/View/ListItem.js]: cool, I have made the changes :)

**Review by kirsty-unosquare — APPROVED** [review]

**Comment by kirsty-unosquare** [comment]
@lee-cheung approved the changes. But wait until @William-Unosquare  has done a quick review. A lot of files to look over and might have missed something.

**Review by William-Unosquare — CHANGES_REQUESTED** [review]
Mostly looks grand. Just fix up the name of the style prop as discussed on slack.

**Comment by lee-cheung** [comment]
@William-Unosquare  thanks for the review. Please could you approve?

**Review by William-Unosquare — APPROVED** [review]
Rename customStyle to style, but approving so as to not block you

**Pull request merged**
